python/lsst/ctrl/oods/oods_csc.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, name, initial_state=salobj.State.STANDBY):
         self.task_list = None
 
         ingester_config = self.config["ingester"]
-        self.ingester = FileIngester(ingester_config)
+        self.ingester = FileIngester(ingester_config, self)
```

Comment:
If I got it right, you're creating an instance of `FileIngester` using a partially initialized object. Can we have a comment pointing that out and briefly explaining why it's okay? (I assume `FileIngester` does not need a `cache_cleaner` being defined.)